### PR TITLE
Added program calculating prime factors of a given number.

### PR DESCRIPTION
**Pull request by zomsik**

### CODE REVIEW

File: Misc/primefactors.cpp [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+                        n/=k;
+                }
+                ++k;
+        }
+
+        if(n>1)
+               cout<<n;
+        cout<<endl;
+
+        system("pause");
```
Comment (yashasvi-goel) [inline Misc/primefactors.cpp]: Why is this needed?

File: Misc/primefactors.cpp [hunk 2]
```diff
@@ -0,0 +1,37 @@
+#include<iostream>
```
Comment (yashasvi-goel) [inline Misc/primefactors.cpp]: Add author Info

File: Misc/primefactors.cpp (hunk 1 above)
Comment (zomsik) [reply Misc/primefactors.cpp]: The project has name: basic C programs, so I added this because it may be useful to someone.

Comment (yashasvi-goel) [reply Misc/primefactors.cpp]: I am asking about line #35 . It is highly OS dependent. please comment it out

Comment (zomsik) [reply Misc/primefactors.cpp]: I thought about it and you are right. I have made a new commit which deleted this line. I'm new in programming so I did not think about the correctness of the code on others OS.

**Comment by yashasvi-goel** [comment]
@all-contributors please add @zomsik  for code
